transcriptic/commands.py

```diff
@@ -1290,11 +1349,11 @@ def parse_valid_org(indx):
 
 
 def get_project_id(api, name):
```

Comment:
I am assuming we will add tests for this method, later.

Reply:
Also, do we really need this method?

Reply:
Yea.. its mostly an internal helper/util method. I'll add a test since I'm at it